app/components/OrganigramInput/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useCallback } from 'react';
+import { _cs } from '@togglecorp/fujs';
+import { PartialForm } from '@togglecorp/toggle-form';
+import OrgTree from 'react-org-tree';
+import {
+    Card,
+} from '@the-deep/deep-ui';
+import styles from './styles.css';
+
+import { OrganigramValue, OrganigramDatum } from '#types/newAnalyticalFramework';
+
+interface Props<N extends string> {
+    className?: string;
+    name: N;
+    value: OrganigramValue | null | undefined;
+    onChange: (value: OrganigramValue | undefined, name: N) => void,
+    options: PartialForm<OrganigramDatum> | null | undefined;
+    readOnly?: boolean;
+    disabled?: boolean;
+}
+
+function OrganigramInput<N extends string>(props: Props<N>) {
+    const {
+        className,
+        name,
+        value,
+        onChange,
+        disabled,
+        readOnly,
+        options,
+    } = props;
+
+    const handleClick = useCallback((_: Event, data: OrganigramDatum) => {
+        if (value?.some((v) => v === data.clientId)) {
+            onChange(value?.filter((v) => v !== data.clientId), name);
+        } else {
+            onChange(([] as string[]).concat(value ?? [], data.clientId), name);
+        }
+    }, [value, onChange, name]);
+
+    const isSelected = useCallback((d: OrganigramDatum) => (
+        value?.some((v) => v === d.clientId)
+    ), [value]);
```

Comment:
Shouldn't this just be a `value?.find` ?

Reply:
boolean value needs to be returned so array.some is more semantic here.